src/api/operations/access.fragments.graphql

```diff
@@ -92,6 +93,8 @@ fragment ProjectAccessFields on ProjectAccess {
   canDeleteAssessments
   canEditProjectDetails
   canManageInventory
```

Comment:
We can stop resolving this since it's deprecated

Reply:
Whoops I just meant removing this line from this file, not from the schema itself. See https://github.com/greenriver/hmis-warehouse/pull/4836/files#r1808731274